include/benchmark/complexity.h

```diff
@@ -19,24 +37,34 @@ enum BigO {
   oAuto
 };
 
-inline std::string GetBigO(BigO complexity) {
-  switch (complexity) {
-    case oN:
-      return "* N";
-    case oNSquared:
-      return "* N**2";
-    case oNCubed:
-      return "* N**3";
-    case oLogN:
-      return "* lgN";
-    case oNLogN:
-      return "* NlgN";
-    case o1:
-      return "* 1";
-    default:
-      return "";
-  }
-}
+// This data structure will contain the result returned by MinimalLeastSq
+//   - coef        : Estimated coeficient for the high-order term as
+//                   interpolated from data.
+//   - rms         : Normalized Root Mean Squared Error.
+//   - complexity  : Scalability form (e.g. oN, oNLogN). In case a scalability
+//                   form has been provided to MinimalLeastSq this will return
+//                   the same value. In case BigO::oAuto has been selected, this
+//                   parameter will return the best fitting curve detected.
+
+struct LeastSq {
```

Comment:
thinking about it more, these shouldn't be in this header. we want to keep the public api (under include/benchmark) as clean as possible and users would never need these.

as such, can you move this into src/ and add the enum definition into benchmark_api.h where the Complexity method is defined?